src/Nav.js

```diff
@@ -18,6 +18,25 @@ const Nav = React.createClass({
     justified: React.PropTypes.bool,
     onSelect: React.PropTypes.func,
     collapsible: React.PropTypes.bool,
+    /**
+     * CSS classes for the wrapper `nav` element
+     * If *ulClassName* is not provided,
+     * then *className* is also applied to the inner `ul` element
+     * (for compatibility with old codebase)
+     */
```

Comment:
I think deprecating this functionality in favor of only applying it to the nav component is a safer default. If people want to explicitly set the ul className then do so explicitly.